10x/count/count.wdl

```diff
@@ -1126,17 +1188,23 @@ task filter_barcodes {
     stage_path=/cellranger/mro/stages/counter/filter_barcodes
     stage_phase=main
     /martian/adapters/python/martian_shell.py $stage_path $stage_phase . . run_file_whatever
+
+    # tar the filtered_matrices_mex folder so it can be extracted from the task
+    tar -cf filtered_matrices_mex.tar filtered_matrices_mex
```

Comment:
Maybe compress this. I think the mtx file is stored in plain text and it probably compresses really well.